diff -Naur sysd/etc/systemd/networkd.conf yuki/etc/systemd/networkd.conf
--- sysd/etc/systemd/networkd.conf	2023-07-28 12:13:02.000000000 +0300
+++ yuki/etc/systemd/networkd.conf	2023-08-06 19:06:40.999724934 +0300
@@ -13,7 +13,7 @@
 # See networkd.conf(5) for details.
 
 [Network]
-#SpeedMeter=no
+SpeedMeter=yes
 #SpeedMeterIntervalSec=10sec
 #ManageForeignRoutingPolicyRules=yes
 #ManageForeignRoutes=yes
diff -Naur sysd/etc/systemd/resolved.conf yuki/etc/systemd/resolved.conf
--- sysd/etc/systemd/resolved.conf	2023-07-28 12:13:02.000000000 +0300
+++ yuki/etc/systemd/resolved.conf	2023-08-06 19:22:20.391974239 +0300
@@ -19,11 +19,11 @@
 # Cloudflare: 1.1.1.1#cloudflare-dns.com 1.0.0.1#cloudflare-dns.com 2606:4700:4700::1111#cloudflare-dns.com 2606:4700:4700::1001#cloudflare-dns.com
 # Google:     8.8.8.8#dns.google 8.8.4.4#dns.google 2001:4860:4860::8888#dns.google 2001:4860:4860::8844#dns.google
 # Quad9:      9.9.9.9#dns.quad9.net 149.112.112.112#dns.quad9.net 2620:fe::fe#dns.quad9.net 2620:fe::9#dns.quad9.net
-#DNS=
+DNS=9.9.9.9#dns.quad9.net
 #FallbackDNS=1.1.1.1#cloudflare-dns.com 9.9.9.9#dns.quad9.net 8.8.8.8#dns.google 2606:4700:4700::1111#cloudflare-dns.com 2620:fe::9#dns.quad9.net 2001:4860:4860::8888#dns.google
 #Domains=
-#DNSSEC=no
-#DNSOverTLS=no
+DNSSEC=yes
+DNSOverTLS=yes
 #MulticastDNS=yes
 #LLMNR=yes
 #Cache=yes
